### PR TITLE
ACMS-1990: Acquia CMS Toolbar changes as per Acquia CMS Gin theme.

### PR DESCRIPTION
**Pull request by deepakmishra2**
**Motivation**

Fixes #ACMS-1990

**Proposed changes**

**Alternatives considered**

**Testing steps**

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: modules/acquia_cms_toolbar/acquia_cms_toolbar.libraries.yml [hunk 1]
```diff
@@ -1,8 +1,3 @@
-styling:
-  version: VERSION
-  css:
-    theme:
-      css/acquia_cms_toolbar.css: {}
```
Comment (rajeshreeputra) [inline modules/acquia_cms_toolbar/acquia_cms_toolbar.libraries.yml:5]: This is required for the below:
- Environment specific background color to environment variable.
- Admin menu styles.

File: modules/acquia_cms_toolbar/css/acquia_cms_toolbar.css [hunk 2]
```diff
@@ -1,671 +0,0 @@
-.acquia-cms-toolbar {
```
Comment (rajeshreeputra) [inline modules/acquia_cms_toolbar/css/acquia_cms_toolbar.css:1]: Include style for below:
- Environment specific background color to environment variable.
- Admin menu styles.

**Review by rajeshreeputra — COMMENTED** [review]
Please check my comment.

File: modules/acquia_cms_common/acquia_cms_common.install [hunk 3]
```diff
@@ -34,6 +34,10 @@ function acquia_cms_common_install($is_syncing) {
     $system_theme_config->set('default', 'olivero')->save();
     $system_theme_config->set('admin', "gin")->save();
 
+    // Enable gin_toolbar module.
+    $module_installer = \Drupal::service('module_installer');
```
Comment (apathak18) [inline modules/acquia_cms_common/acquia_cms_common.install:38]: @deepakmishra2 Check line#63 using of module_installer service is duplicate within hook_install

Comment (deepakmishra2) [reply modules/acquia_cms_common/acquia_cms_common.install:38]: done

File: modules/acquia_cms_toolbar/css/acquia_cms_toolbar.css (hunk 2 above)
Comment (deepakmishra2) [reply modules/acquia_cms_toolbar/css/acquia_cms_toolbar.css]: As per discussion in DSM included styles from Environment specific background color to environment variable but admin menu styles are removed.

File: modules/acquia_cms_toolbar/acquia_cms_toolbar.libraries.yml (hunk 1 above)
Comment (deepakmishra2) [reply modules/acquia_cms_toolbar/acquia_cms_toolbar.libraries.yml]: As per discussion in DSM included styles from Environment specific background color to environment variable but admin menu styles are removed.

**Comment by rajeshreeputra** [comment]
Please remove Gin theme from [composer.json](https://github.com/acquia/acquia_cms/blob/ACMS-1990-1/modules/acquia_cms_common/composer.json#L20) file, as it should come from starterkit for new customers, for existing customer steps should be provided. Gin should not be a tightly-coupled with Acquia CMS.
@vishalkhode1 @vipin-mittal-acquia what is your opinion on this?

File: modules/acquia_cms_common/acquia_cms_common.install [hunk 4]
```diff
@@ -15,8 +15,9 @@ use Drupal\workbench_email\Entity\Template;
  * Implements hook_install().
  */
 function acquia_cms_common_install($is_syncing) {
-  $module_handler = \Drupal::moduleHandler();
   if (!$is_syncing) {
+    $module_handler = \Drupal::moduleHandler();
+    $moduleInstaller = \Drupal::service('module_installer');
```
Comment (apathak18) [inline modules/acquia_cms_common/acquia_cms_common.install:20]: @rajeshreeputra Use snake_case for .install .module files 

cc: @vishalkhode1 @chandan-singh7929

File: modules/acquia_cms_common/acquia_cms_common.install [hunk 5]
```diff
@@ -27,12 +28,27 @@ function acquia_cms_common_install($is_syncing) {
...
+    \Drupal::service('theme_installer')->install($themesInstall);
+    $systemThemeConfig = \Drupal::configFactory()->getEditable('system.theme');
+    $systemThemeConfig->set('default', 'olivero')->save();
+    if (in_array("gin", $themesInstall)) {
+      $systemThemeConfig->set('admin', "gin")->save();
+      // Add colors to gin.
+      $ginSettings = \Drupal::configFactory()->getEditable('gin.settings');
+      $ginSettings->set('accent_color', '#096b92');
+      $ginSettings->set('focus_color', '#168ec4');
+      $ginSettings->set('classic_toolbar', 'classic');
```
Comment (apathak18) [inline modules/acquia_cms_common/acquia_cms_common.install:47]: @rajeshreeputra As per the discussion remove this implementation

cc: @chandan-singh7929 @vishalkhode1

Comment (rajeshreeputra) [reply modules/acquia_cms_common/acquia_cms_common.install]: Done.

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**